src/components/DpVideoPlayer/DpVideoPlayer.vue

```diff
@@ -37,6 +47,12 @@ export default {
       default: false
     },
 
+    dataCy: {
+      type: String,
+      required: false,
+      default: 'videoplayer'
```

Comment:
videoPlayer*

Reply:
Done in [1fe4d4a](https://github.com/demos-europe/demosplan-ui/pull/1005/commits/1fe4d4a2745b43094c07226874423d294d465995)